src/components/bookmarkButton/BookmarkButton.spec.js

```diff
@@ -0,0 +1,11 @@
+import BookmarkButton from './BookmarkButton'
+import { render, screen } from '@testing-library/react'
+
+describe('BookmarkButton', () => {
+  it('renders', () => {
```

Comment:
Here I would add the test whether `onClickBookmarked` will be called when clicking the button.

Reply:
Added.